app_config/wallbox_modbus_registers.yaml

```diff
@@ -65,6 +65,10 @@ charging_state: &charging 1
 # Connected and waiting for car demand; sometimes shortly goes to this status when action = start
 waiting_state: &waiting 2
 
+# Connected and waiting for next schedule; this occurs when a charging is scheduled via the Wallbox app.
```

Comment:
[...] when a charging **session** is scheduled [...]

Reply:
OK